runtime/compiler/codegen/J9CodeGenerator.cpp

```diff
@@ -633,9 +632,7 @@ J9::CodeGenerator::lowerCompressedRefs(
    if (TR::Compiler->om.compressedReferenceShiftOffset() > 0)
       shftOffset = TR::Node::create(loadOrStoreNode, TR::iconst, 0, TR::Compiler->om.compressedReferenceShiftOffset());
 
-   static char *pEnv = feGetEnv("TR_disableLowHeapMem");
-   if (pEnv)
-      isLowMem = false;
+   bool isLowMem = feGetEnv("TR_disableLowHeapMem") ? false : true;
```

Comment:
I think we should fold this as `true` and eliminate the variable along with the `feGetEnv` call. Note that `feGetEnv` will get called every time though this path now. This is not something we typically want as that call is quite expensive. This is why we typically make such calls and assign to `static` variables (so it only gets called once).

Reply:
330ca59